src/cacao.ts

```diff
@@ -78,6 +84,45 @@ export namespace Cacao {
 
     return cacao
   }
+
+  export function verify(cacao: Cacao): VerificationResult {
+    if (cacao.h.t === 'eip4361-eip191') {
+      return verifyEIP191Signature(cacao)
+    }
+    throw new Error('Unsupported CACAO signature type')
+  }
+
+  export function verifyEIP191Signature(cacao: Cacao): VerificationResult {
+    try {
+      if (!cacao.s) {
+        throw new Error(`CACAO does not have a signature`)
+      }
+
+      if (cacao.p.iat > Date.now() || cacao.p.nbf > Date.now()) {
```

Comment:
We should use an [`atTime`](https://github.com/ceramicnetwork/js-did/blob/f20e486a4cb1edd62093a63f9f957da5946b87f9/src/did.ts#L46) option similar to js-did when doing the verification of the timestamp. The time we are trying to verify this Cacao might be in the past so we can't rely on `Date.now()`.

Reply:
Ah thats good, will do